### PR TITLE
Improve jsdocs

### PR DESCRIPTION
**Pull request by thms-rmb**
Just added some documentation for the variable `rootUrl` and the function `parseJSON()`.

### CODE REVIEW

File: src/fetch.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Uses a Response object to create a good old JavaScript object.
```
Comment (esphen) [inline src/fetch.js]: You mean "Plain old JavaScript object"? :)

https://en.wikipedia.org/wiki/Plain_old_Java_object

Comment (thms-rmb) [inline src/fetch.js]: Hehe, I guess!

Maybe it's better if we put `@ignore` over them instead? Or both? Is there some way to use JSDoc to get the editor to understand the code better, while still keeping them from appearing on the documentation site?

Comment (esphen) [inline src/fetch.js]: Not sure, I don't use an editor with jsdoc support. You'll have to try out what works best!

The only thing is that these should not appear in the generated documentation, apart from that, knock yourself out

**Comment by esphen** [comment]
We have a conflict on a doc file now! Another reason why we should not be commiting them into the repo and instead be generating in travis

**Comment by esphen** [comment]
🎉

**Pull request merged**